clients/go/zpu/cmd/tools/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"flag"
+	"log"
+
```

Comment:
let's remove the new lines so from the standard package blocks so they're properly sorted.